handlers/holiday-stop-processor/src/main/scala/com/gu/holidaystopprocessor/Salesforce.scala

```diff
@@ -12,23 +12,18 @@ import scalaz.{-\/, \/-}
 import com.gu.holiday_stops.{OverallFailure, SalesforceHolidayWriteError}
 
 object Salesforce {
-
-  def holidayStopRequests(sfCredentials: SFAuthConfig)(productNamePrefix: ProductName, processDate: LocalDate): Either[OverallFailure, List[HolidayStopRequestsDetail]] = {
+  def holidayStopRequests(sfCredentials: SFAuthConfig)(productKey: ProductRatePlanKey, processDate: LocalDate): Either[OverallFailure, List[HolidayStopRequestsDetail]] = {
```

Comment:
```suggestion
  def holidayStopRequests(sfCredentials: SFAuthConfig)(productRatePlanKey: ProductRatePlanKey, processDate: LocalDate): Either[OverallFailure, List[HolidayStopRequestsDetail]] = {
```
And perhaps give a comment to explain what a `ProductRatePlanKey` is where it's defined